source/codegen/metadata/nidigitalpattern/config.py

```diff
@@ -46,38 +31,9 @@
         }
     },
     'linux_rt_support': False,
-    'metadata_version': '2.0',
-    'module_name': 'nidigitalpattern',
```

Comment:
Can you fix 'module_name' back to nidigital? It'll be the same fix done for namespace_component in the hapigen global_config file(s).